moredata/utils/osm_downloader.py

```diff
@@ -53,10 +53,10 @@ def get_places(self, place_name, key, value, query=None, tags=("name","geom")):
             properties = data["properties"]
             if "tags" in properties.keys():
                 tags = properties["tags"]
-
                 for item in tags.items():
                     key, value = item[0], item[1]
-                    properties[key] = value
+                    properties['value'] = value
+                    properties['key'] = key
             
```

Comment:
After assigning the variables, you would create the variables related to tags, so you can do: `properties[k]=v`